api/src/main/java/org/openmrs/Obs.java

```diff
@@ -435,6 +458,8 @@ public void setGroupMembers(Set<Obs> groupMembers) {
 	 * @param member Obs to add to this group
 	 * @see #setGroupMembers(Set)
 	 * @see #getGroupMembers()
+	 * @should return true when a new obs is added as a member
```

Comment:
Correct, you can rephrase it, it should be something like '@should mark the obs as dirty when a new member is added'